Math_Quiz_App.py

```diff
@@ -122,12 +122,17 @@ def submit_difference_answer():
             response2.place(x=20, y=120)
             difference_entry.delete(0, END)
     else:
-        next_question_button.destroy()
-        subtraction_question.destroy()
-        difference_entry.destroy()
-        submit_difference_button.destroy()
-        end_label = Label(root, text="You are done!", font=("calibri", 20))
-        end_label.pack(side=TOP)
+        response2 = Label(root, text="Sorry, your answer is wrong. Please try again")
```

Comment:
Hi Charlie, could you optimize your code better, such as using a class to manage addition, subtraction, multiplication, division.

Reply:
thank you for you suggestion, it is a good suggestion. I will consider it in other pr, this pr is only to focus on fixing the subtraction feature. can you approve it first, if you still have some concerns, please let me know.

Reply:
Sure!